AssignedTo.h

```diff
@@ -0,0 +1,22 @@
+//
+//  AssignedTo.h
+//  FilterSearchAttempt
+//
+//  Created by Lizzy Randall on 5/5/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface AssignedTo : NSObject {
+    NSString *name;
+    BOOL checked;
+}
+
+@property (nonatomic, copy) NSString *name;
+@property (nonatomic) BOOL checked;
+
++ (id)constructAssignedTo:(NSString*)name checked:(BOOL)checked;
```

Comment:
naming conventions for constructors are a little different. You skip the construct and just start with the type that will be returned. If there are parameters you add With<parameter name> to the method name.
e.g. -(instancetype)assignedToWithName:(NSString *)name checked:(BOOL)checked;
you will notice that I changed the return type. instancetype is a special type for constructors mainly. It is similar to id in that you don't have to explicitly state the type being returned ( which is a good idea in case you refactor the class or make a subclass but don't want to override the constructor) but using instance type means the IDE and compiler still now to throw errors if you try to use methods the return object doesn't have eg [[AssignedTo assignedToWithName:@"myName" checked:YES] objectForKey:@"blue"]; would throw an error with instancetype but wouldn't if you used id.